core/templates/pages/exploration-editor-page/history-tab/history-tab.component.html

```diff
@@ -18,48 +18,50 @@ <h3 class="history-card-header-text">History</h3>
         <i class="fa fa-search"></i>
       </div>
       <div class="history-table-header">
-        <div class="history-table-header-version">
+        <div tabindex="0" class="history-table-header-version">
           <span>Version No.</span>
         </div>
-        <div class="history-table-header-message">
+        <div tabindex="0" class="history-table-header-message">
           <span>Notes</span>
         </div>
-        <div class="history-table-header-profile">
+        <div tabindex="0" class="history-table-header-profile">
           <span>User</span>
         </div>
-        <div class="history-table-header-date" (click)="reverseDateOrder()">
+        <div tabindex="0" class="history-table-header-date" (click)="reverseDateOrder()" (keydown.enter)="reverseDateOrder()">
           <span>Date</span>
         </div>
       </div>
-      <div class="display-container">
+      <div tabindex="0" class="display-container">
         <span>Displaying {{ startingIndex }}-{{ endIndex }} of {{ totalExplorationVersionMetadata.length }}</span>
       </div>
       <div *ngFor="let metadata of explorationVersionMetadata; index as index" class="history-list-item e2e-test-history-list-item">
-        <div class="history-table-index">
+        <div tabindex="0" class="history-table-index">
           {{ metadata.versionNumber }}.
         </div>
-        <div class="history-table-message e2e-test-history-table-message">
+        <div tabindex="0" class="history-table-message e2e-test-history-table-message">
           {{ metadata.commitMessage }}
         </div>
         <div class="history-table-profile">
           <strong><profile-link-text [username]="metadata.committerId"></profile-link-text></strong>
         </div>
-        <div class="history-table-version" title="{{ metadata.tooltipText }}">
+        <div tabindex="0" class="history-table-version" title="{{ metadata.tooltipText }}">
           <em class="e2e-test-history-tab-commit-date">{{ metadata.createdOnMsecsStr }}</em>
         </div>
         <div class="history-table-options e2e-test-history-list-options">
-          <div class="dropdown">
-            <div type="button" id="{{ 'dropdownMenuButton-' + index }}" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+          <div class="dropdown" [ngClass]="{ show: isDropdownVisible }" >
+            <div aria-label="More option" type="button" id="{{ 'dropdownMenuButton-' + index }}" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" [attr.aria-expanded]="isDropdownVisible.toString()" (keydown)="moreOptionButton($event, dropdownMenuButton)" tabindex="0" #dropdownMenuButton>
```

Comment:
Why do we do "toString" on `isDropdownVisible`? It appears to be a boolean variable.